src/coreclr/jit/utils.h

```diff
@@ -759,7 +759,8 @@ class CritSecHolder
 
 namespace MagicDivide
 {
-uint32_t GetUnsigned32Magic(uint32_t d, bool* increment /*out*/, int* preShift /*out*/, int* postShift /*out*/);
+uint32_t GetUnsigned32Magic(
+    uint32_t d, bool* increment /*out*/, int* preShift /*out*/, int* postShift /*out*/, unsigned bits = 32);
```

Comment:
`bits` is always passed as a parameter, so I would recommend to not have it default but always passed by the caller. In `GetUnsigned32Magic()` you could add `assert(bits <= 32)` and in `GetUnsigned64Magic()` you could add `assert(bits <= 64)`.